projects/plugins/jetpack/extensions/blocks/subscriptions/email-preview.js

```diff
@@ -112,3 +117,151 @@ export default function EmailPreview( { isModalOpen, closeModal } ) {
 		</>
 	);
 }
+
+const devices = [
+	{ name: 'desktop', icon: desktop, label: __( 'Desktop', 'jetpack' ), width: '100%' },
+	{ name: 'tablet', icon: tablet, label: __( 'Tablet', 'jetpack' ), width: '768px' },
+	{ name: 'mobile', icon: mobile, label: __( 'Mobile', 'jetpack' ), width: '360px' },
+];
+
+const DevicePicker = ( { selectedDevice, setSelectedDevice } ) => (
+	<ButtonGroup>
+		{ devices.map( device => (
+			<Button
```

Comment:
I think we should add a label to these buttons for improved accessibilty. See label https://developer.wordpress.org/block-editor/reference-guides/components/button/